src/OSPSuite.Infrastructure.Import/Core/InvalidFileException.cs

```diff
@@ -1,11 +1,12 @@
 ﻿using OSPSuite.Assets;
 using System;
+using OSPSuite.Utility.Extensions;
 
 namespace OSPSuite.Infrastructure.Import.Core
 {
    public class InvalidFileException : Exception
```

Comment:
please derive from `OSPSuiteException`

Reply:
also other importer exceptions should be derived from OSPSuiteException
![grafik](https://user-images.githubusercontent.com/25061876/125804596-2e1f58be-c3f0-41b0-bf7e-b9ce34f0d886.png)

Reply:
done. I also realized there are 3 more exceptions that derive directly from Exception. Should I change those too, or are they on purpose?
![image](https://user-images.githubusercontent.com/37107428/125900962-7db8d0fb-c47a-455a-af4f-849dab8e1c71.png)